_pages/security-events.md

```diff
@@ -82,16 +97,25 @@ JWTs must be signed by the client application's private key using **RS256**, the
 
 * **events**
 
-  An object containing an event, keyed by event type. See [Supported Events](#supported-events) for possible keys. An event should contain a **subject** object, with the following keys:
+  An object containing an event, keyed by event type. See [Supported Events](#supported-events) for possible keys.
+
+  The event (the value) contains:
+
+  * **subject** (required)
+      An event should contain a **subject** object, with the following keys:
+
+      * **subject_type**
+        Must be **iss_sub**, this indicates the **sub** is the subject provided by the original issuer (login.gov)
+
+      * **iss**
+        This is login.gov's issuer, the root URL for login.gov. In the agency integration environment, this is `https://idp.int.identitysandbox.gov`
 
-  * **subject_type**
-    Must be **iss_sub**, this indicates the **sub** is the subject provided by the original issuer (login.gov)
+      * **sub**
+        The UUID identifying the user. This is provided as the `sub` inside the `id_token` JWT in the [OpenID Token endpoint]({{site.baseurl}}/#token-response).
 
-  * **iss**
-    This is login.gov's issuer, the root URL for login.gov. In the agency integration environment, this is `https://idp.int.identitysandbox.gov`
+  * **event_at**
```

Comment:
I almost went with `occurred_at` but I kept spelling "occurred" wrong so I figured we could maybe prevent future typos this way. However, the name is not very descriptive, open to thoughts here

Reply:
I think `event_at` is ok, a more verb-y option could be something like `discovered_at`?

Reply:
So my use case is, suppose the fraud happened 2 weeks ago, but we figured it out now and reported it now. I'm looking to capture the 2 weeks ago timestamp, but I am worried `discovered_at` would lead people to submit "now" since it's when they discovered the event?

Reply:
Ohhh, right right. `event_at` I think could be too ambiguous. Maybe `happened_at` or `affected_at`?

Reply:
Yeah I like `happened_at` out of the two of those, but I'm warming back up to  `occurred_at`

Reply:
I think either works 👍🏼

Reply:
I also can't spell occurred, but I think it captures the meaning the best and am a fan.